app/src/main/java/com/codesoom/assignment/application/TaskApplicationService.java

```diff
@@ -0,0 +1,68 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.Task;
+import com.codesoom.assignment.optional.Empty;
+
+import java.util.*;
+
+public class TaskApplicationService {
+    private final Map<Long, Task> taskMap = new HashMap<>();
+    private Long lastId = 1L;
+
+    public List<Task> getAllTasks() {
+        ArrayList<Task> allTasks = new ArrayList<>(taskMap.values());
```

Comment:
`List` 인터페이스로 할당해주는 것도 괜찮을 것 같습니다.
```suggestion
        List<Task> allTasks = new ArrayList<>(taskMap.values());
```